pom.xml

```diff
@@ -69,55 +74,35 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
-    <profiles>
-        <profile>
-            <id>generate-binaries-profile</id>
-            <activation>
-                <property>
-                    <name>env.SKIP_DURABLE_TASK_BINARY_GENERATION</name>
-                    <value>!true</value>
-                </property>
-            </activation>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-jar-plugin</artifactId>
-                        <version>3.1.2</version>
-                        <configuration>
-                            <archive>
-                                <manifest>
-                                    <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
-                                    <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
-                                </manifest>
-                            </archive>
-                        </configuration>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>exec-maven-plugin</artifactId>
-                        <version>1.6.0</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.2</version>
+                <executions>
+                    <execution>
+                        <id>extract-binaries</id>
```

Comment:
Why bother with all this? You can simply declare a dep on that JAR and it will be bundled inside the plugin and its resources accessible. (Though I would then suggest making the upstream project package binaries into a reasonable package prefix.)

Reply:
So I did all this to basically rename the binary file. The binary name is `durable_task_monitor_VER_ARCH_BITS`. I was replacing the VER with the `durable_task` VER. 
I believe I was thinking I wanted to people to know which version of durable-task was running the binary. That's probably not necessary since users will just look at their jenkins instance to figure out what's going on.

Reply:
Oh yes, there was also the question of how to get the version of `lib-durable-task` since it's not a plugin and doesn't really have any java code. I could parse the pom, I guess.

Reply:
> how to get the version of `lib-durable-task`

Why would you need to?

```java
URL bin = BourneShellScript.class.getResource("/io/jenkins/plugins/lib-durable-task/durable_task_monitor_amd64");
```

Am I missing something?

Reply:
no you're not...i totally got lost in the weeds for this 👀

Reply:
and of course I just belatedly realized that I can probably use `getResourceAsStream` and rename the binary there as well.

Reply:
Resolved in c658ad81067e362c9fb0c9d51763d3232a97a6f2
Completely removed this section of the POM and use the package resource in `BourneShellScript`. No more messing with versions and filenames